files/chef-marketplace-ctl-commands/show_instance_id.rb

```diff
@@ -0,0 +1,20 @@
+require "ohai"
+
+add_command_under_category "show-instance-id", "Configuration", "Show the cloud instance ID. Required for first time setup authorization.", 2 do
+  puts "Looking up current Instance ID..."
+
+  system = Ohai::System.new.all_plugins
```

Comment:
`system` is a "reserved" keyword because it's in the [`kernel` module](https://ruby-doc.org/core-2.5.1/Kernel.html#method-i-system). Overloading it is generally not a good idea.